package/gluon-status-page-api/src/neighbours-batadv.c

```diff
@@ -0,0 +1,65 @@
+#include <stdio.h>
+#include <string.h>
+#include <unistd.h>
+#include <json.h>
```

Comment:
The recommended include for json-c is `<json-c/json.h>`, the `<json.h>` include is deprecated and will be removed from json-c in the future.